src/sql/workbench/contrib/query/browser/gridPanel.ts

```diff
@@ -435,11 +435,12 @@ export abstract class GridTableBase<T> extends Disposable implements IView {
 			this.container.appendChild(actionBarContainer);
 		}
 
-		let tableContainer = document.createElement('div');
-		tableContainer.style.display = 'inline-block';
-		tableContainer.style.width = `calc(100% - ${ACTIONBAR_WIDTH}px)`;
+		this.tableContainer = document.createElement('div');
+		this.tableContainer.className = 'grid-panel';
```

Comment:
Moved the "grid-panel" class from the parent container to this one. The grid-panel class was messing with some styling in the charts, like causing buttons to have text overlapping the icons.

Reply:
Other components under the parentContainer already had the grid-panel class, so not much change here.